java/pom.xml

```diff
@@ -767,45 +774,99 @@
     </profile>
 
     <profile>
-      <id>error-prone-jdk8</id>
-      <!--
-           Do not activate Error Prone while running with Eclipse/M2E as it causes incompatibilities
-           with other annotation processors.
-           See https://github.com/jbosstools/m2e-apt/issues/62 for details
-      -->
+      <id>error-prone-checkerframework-jdk8</id>
```

Comment:
IMO, we don't bother enabling this for JDK8. In fact, I'd be OK with dropping error-prone for JDK8 as well, and only enabling these checks only on the latest LTS.